spring/src/main/java/org/axonframework/spring/config/SpringAxonAutoConfigurer.java

```diff
@@ -19,6 +19,7 @@
 import org.axonframework.commandhandling.CommandBus;
```

Comment:
I expect an additional test case in the `SpringAxonAutoConfigurerTest` which verifies the new `Aggregate` value `cache` to be picked up, resulting in an `CachingEventSourcingRepository` for a given Aggregate.